sklearn_numba_dpex/common/tests/test_kernels.py

```diff
@@ -1,16 +1,16 @@
-import dpctl
 import dpctl.tensor as dpt
 import numpy as np
 import pytest
 from sklearn.utils._testing import assert_allclose
 
 from sklearn_numba_dpex.common.kernels import (
     make_argmin_reduction_1d_kernel,
-    make_sum_reduction_2d_axis1_kernel,
+    make_sum_reduction_2d_kernel,
 )
 from sklearn_numba_dpex.testing.config import float_dtype_params
 
 
+@pytest.mark.parametrize("axis", [0, 1])
 @pytest.mark.parametrize("work_group_size", [2, 4, 8, "max"])
```

Comment:
The tests work with wgs=1 so apparently yes we can keep the test.

Reply:
In fact it passes on gpu but not on cpu. I don't think it's worth debugging that :thinking:

Reply:
At least let's add an inline comment to document the problem. If you can find a minimal reproducer that would be great to get it fixed upstream so that we don't encounter the same bug again 6 months later down the road.

Reply:
Was this comment addressed?

Reply:
yes, kernel now raises an error for `work_group_size == 1` https://github.com/soda-inria/sklearn-numba-dpex/pull/80/files#diff-326118b7b149309370de9833d674e45b523445573056b66b14daed1793f00fe2R320